### PR TITLE
feat(metrics): add click event for 2fa setup totp code submit

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

**Review by LZoog — APPROVED** [review]
LGTM, but should we add this to `InlineTotpSetup` too?

**Comment by chenba** [comment]
> LGTM, but should we add this to `InlineTotpSetup` too?

Sounds good.  Thank you for the reminder.

**Comment by chenba** [comment]
@LZoog the inline setup is quite different so I re-request a review.  Thanks!

File: packages/fxa-settings/src/components/FormVerifyCode/index.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { FtlMsg } from 'fxa-react/lib/utils';
 import { logViewEvent } from '../../lib/metrics';
 import { REACT_ENTRYPOINT } from '../../constants';
 import { useFtlMsgResolver } from '../../models';
+import { GetDataTrioGleanData } from '../GetDataTrio';
```
Comment (LZoog) [inline packages/fxa-settings/src/components/FormVerifyCode/index.tsx:12]: Unused import.

File: packages/fxa-settings/src/lib/types.ts [hunk 2]
```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+export type GleanClickEventDataAttrs = {
```
Comment (LZoog) [inline packages/fxa-settings/src/lib/types.ts:5]: 👍

**Review by LZoog — APPROVED** [review]
Tested locally, :shipit:

**Pull request merged**